src/store/store.ts

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from "@reduxjs/toolkit";
+import { feedBackGridItemReducer } from '../app/pages/HomePage/components/TestimonialPart/components/FeedBackGridItem/feedBackGridItem.slice'
+
+export const store = configureStore ( {
+        reducer: {
+         'feedBackGridItem': feedBackGridItemReducer,
+        }
+ });
+ export type RootState = ReturnType<typeof store.getState>;
```

Comment:
but all the code that is connected with Redux is correct)
just needed to create another slice